plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -398,9 +408,7 @@ instance Flat DefaultFun where
               LessThanInteger       -> 5
               LessThanEqualsInteger -> 6
               EqualsInteger         -> 9
-              Concatenate           -> 10
-              TakeByteString        -> 11
-              DropByteString        -> 12
```

Comment:
Just for the moment we're trying to avoid messing up the existing scripts, since Kenneth is using them. Can you leave take/drop in for now, and we'll remove them in a follow-up?

Reply:
Yeah, sure. I will open a new PR with their removal then.